source/c_softpwm.c

```diff
@@ -0,0 +1,339 @@
+/*
+Copyright (c) 2016 Robert Wolterman
```

Comment:
Go ahead and add your copyright info to the files you've added/touched.  You wrote the code, you should get the credit. :)